src/libs/auth.js

```diff
@@ -17,24 +18,47 @@ import { asyncImportJobStore, authStore, cookieReadyStore, requesterPaysProjectS
 import * as Utils from 'src/libs/utils'
 
 
+export const getOidcConfig = () => {
+  return {
+    authority: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+    client_id: getConfig().oidcClientId,
+    redirect_uri: window.origin,
+    prompt: 'login',
+    scope: 'openid email profile',
+    metadata: {
+      authorization_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+      token_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/token`
+    }
+  }
+}
+
 const getAuthInstance = () => {
-  return window.gapi.auth2.getAuthInstance()
+  return authStore.get().authContext
 }
 
 export const signOut = () => {
   // Note: Ideally, we'd actually clear the Leo proxy cookie here, but there's not currently an endpoint to do that.
   // When IA-2236 is done, add that call here.
   cookieReadyStore.reset()
   sessionStorage.clear()
-  getAuthInstance().signOut()
+  getAuthInstance().removeUser()
+}
+
+export const signIn = includeBillingScope => {
+  if (includeBillingScope) {
```

Comment:
I'm having trouble with the billing scope. Repro steps:

1) Log in to Terra as Hermione, go to Billing page
2) Pick a project Hermione owns. Note the "No Access" by "Billing Account"-- Hermione should have access.

Looks like:
![image](https://user-images.githubusercontent.com/484484/165326352-e0fdae9f-7d47-42d7-9577-df0d2019a561.png)

Should look like:
![image](https://user-images.githubusercontent.com/484484/165326548-b8c1eb4c-d315-435f-a969-2a1fa9f7d77e.png)

3) Try clicking the edit icon. You get prompted to choose your account again, but it still doesn't correct the issue.

Reply:
Damn I thought this worked.. thanks will look.

Reply:
think I fixed this, it wasn't remembering the scope correctly.